chefs_soils.py

```diff
@@ -1,73 +1,1379 @@
+
+print('Loading High Volume Sites list...')
+hvsJson = helper.site_list(CHEFS_HV_FORM_ID, CHEFS_HV_API_KEY)
+#print(hvsJson)
+print('Loading High Volume Sites attributes and headers...')
+hvsAttributes = helper.fetch_columns(CHEFS_HV_FORM_ID, CHEFS_HV_API_KEY)
+# print(hvsAttributes)
+
+# Fetch subscribers list
+print('Loading submission subscribers list...')
+subscribersJson = helper.site_list(CHEFS_MAIL_FORM_ID, CHEFS_MAIL_API_KEY)
+#print(subscribersJson)
+print('Loading submission subscribers attributes and headers...')
+subscribeAttributes = helper.fetch_columns(CHEFS_MAIL_FORM_ID, CHEFS_MAIL_API_KEY)
+# print(subscribeAttributes)
+
+
+print('Creating source site, receiving site records...')
+sourceSites = []
+receivingSites = []
+rcvRegDistDic = {}
+for submission in submissionsJson:
+  #print('Mapping submission data to the source site...')
+  _srcDic = map_source_site(submission)
+  if _srcDic:
+    sourceSites.append(_srcDic)
+
+  #print('Mapping submission data to the receiving site...')  
+  _1rcvDic = map_rcv_1st_rcver(submission)
+  if _1rcvDic:
+    receivingSites.append(_1rcvDic)
+    add_regional_district_dic(_1rcvDic, rcvRegDistDic)
+
+  _2rcvDic = map_rcv_2nd_rcver(submission)
+  if _2rcvDic:
+    receivingSites.append(_2rcvDic)
+    add_regional_district_dic(_2rcvDic, rcvRegDistDic)
+
+  _3rcvDic = map_rcv_3rd_rcver(submission)
+  if _3rcvDic:
+    receivingSites.append(_3rcvDic)
+    add_regional_district_dic(_3rcvDic, rcvRegDistDic)  
+
+print('Creating high volume site records records...')
+hvSites = []
+hvRegDistDic = {}
+for hvs in hvsJson:
+  #print('Mapping hv data to the hv site...')  
+  _hvDic = map_hv_site(hvs)
+  if _hvDic:
+    hvSites.append(_hvDic)
+    add_regional_district_dic(_hvDic, hvRegDistDic)  
+
+
+
+print('Creating soil source site CSV...')
+print('>> current directory:' + os.getcwd())
+with open(SOURCE_CSV_FILE, 'w', encoding='UTF8', newline='') as f:
+  writer = csv.DictWriter(f, fieldnames=SOURCE_SITE_HEADERS)
+  writer.writeheader()
+  writer.writerows(sourceSites)
+
+print('Creating soil receiving site CSV...')
+with open(RECEIVE_CSV_FILE, 'w', encoding='UTF8', newline='') as f:
+  writer = csv.DictWriter(f, fieldnames=RECEIVING_SITE_HEADERS)
+  writer.writeheader()
+  writer.writerows(receivingSites)
+
+print('Creating soil high volume site CSV...')
+with open(HIGH_VOLUME_CSV_FILE, 'w', encoding='UTF8', newline='') as f:
+  writer = csv.DictWriter(f, fieldnames=HV_SITE_HEADERS)
+  writer.writeheader()
+  writer.writerows(hvSites)
+
+
+
+print('Connecting to AGOL GIS...')
+# connect to GIS
+_gis = GIS(MAPHUB_URL, username=MAPHUB_USER, password=MAPHUB_PASS)
+
+print('Updating Soil Relocation Soruce Site CSV...')
+_srcCsvItem = _gis.content.get(SRC_CSV_ID)
+if _srcCsvItem is None:
+  print('Error: Source Site CSV Item ID is invalid!')
+else:
+  _srcCsvUpdateResult = _srcCsvItem.update({}, SOURCE_CSV_FILE)
+  print('Updated Soil Relocation Source Site CSV sucessfully: ' + str(_srcCsvUpdateResult))
```

Comment:
A little typo around `sucessfully` should be `successfully` in multiple places.

Reply:
fixed the typo.